packages/slice-machine/components/ReviewModal/index.tsx

```diff
@@ -49,11 +69,65 @@ const SelectRatingComponent = ({ field, form }: FieldProps) => {
   );
 };
 
-const ReviewModal: React.FunctionComponent<ReviewModalProps> = ({
-  close,
-  isOpen,
-  onSubmit,
-}) => {
+const ReviewModal: React.FunctionComponent<ReviewModalProps> = () => {
+  const { customTypes } = useContext(CustomTypesContext);
+  const libraries = useContext(LibrariesContext);
+  const { isReviewLoading, isLoginModalOpen, hasSendAReview } = useSelector(
+    (store: SliceMachineStoreType) => ({
+      isReviewLoading: isLoading(store, LoadingKeysEnum.REVIEW),
+      isLoginModalOpen: isModalOpen(store, ModalKeysEnum.LOGIN),
+      hasSendAReview: userHasSendAReview(store),
+    })
+  );
+
+  const dispatch = useDispatch();
+  const openLogin = () =>
+    dispatch(modalOpenCreator({ modalKey: ModalKeysEnum.LOGIN }));
+  const startReviewLoading = () =>
+    dispatch(startLoadingActionCreator({ key: LoadingKeysEnum.REVIEW }));
+  const stopReviewLoading = () =>
+    dispatch(stopLoadingActionCreator({ key: LoadingKeysEnum.REVIEW }));
+  const skipReview = () => dispatch(skipReviewCreator());
+  const sendAReview = () => dispatch(sendAReviewCreator());
```

Comment:
I don't know if this is a good idea, because the every custom hooks will call useDispatch, so this hook will be call many times on one component.

If we create custom hooks for every actions creator, we will have to import them as well so it the same right ?

Reply:
Seen together, now it's fixed :)